marathon.user.js

```diff
@@ -940,6 +945,12 @@ async function initGMC() {
         title: "Uncheck if you don't use Disney+",
         default: true,
       },
+      starplus: {
+        type: "checkbox",
+        label: "Star+",
+        title: "Uncheck if you don't use Star+",
+        default: true,
+      },
```

Comment:
If the `disneyplus()` site method works for starplus, then a checkbox is not needed. Like `primevideo` is just an alias of `amazon` so you can do the same with `starplus` and `disneyplus`. That will leave more room in the settings page for unique sites in the future.

Reply:
Oh ok, gonna remove.